src/quo/components/buttons/logout_button/view.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo.components.buttons.logout-button.view
+  (:require
+    [quo.components.buttons.logout-button.style :as style]
+    [quo.components.icon :as icon]
+    [quo.components.markdown.text :as text]
+    [quo.foundations.colors :as colors]
+    [quo.theme :as theme]
```

Comment:
We usually require `quo.theme` as `quo.theme` to prevent confusion with the theme prop.